_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+---
+layout:       pattern
+title:        Lists
+date:         2015-12-21 00:00:00
+
+categories:   patterns
+tags:
+- atomic
+- pattern
+- lists
+
+slug:         lists
+
+url_styles:     "patterns/_lists.scss"
+
+description: Lists are used to group related content in a number of settings and cases.
+
+info: Lists are used to group related content in a number of settings (e.g. ordered, unordered) and cases (e.g. definitions, links, etc.).
+---
+
+<h3 class="hd-6 example-set-hd">List (using different semantics)</h3>
+<div class="example-set">
```

Comment:
Thanks, @cptvitamin. After our talk, your comments, and reflecting, these example fell under the irrelevant/superfluous categories. I've decided to remove them from the doc site rendering (though one could still use the styling to get this affect). Seemed the best way to address things at a larger level.